### DISCUSSION TITLE
ui.number event parameter value differs from object value properties

### DISCUSSION THREAD

**Discussion by CrystalWindSnake** [zauberzeug/nicegui#3110] [Q&A]
### Question

```python
# notify 0 when deleting value
ui.number(value=66, on_change=lambda e: ui.notify(f"{e.value=} {e.sender.value=}"))
```
When the input is cleared, e.value is `0` , expect it to be `None` . this is bug?

**Comment by falkoschindler** [comment]
Thanks, @CrystalWindSnake!
Indeed, this looks like a bug somewhere in `ui.number` or even `ValueElement`. I'll create an issue from this discussion.

**Reply by falkoschindler** [reply]
See issue #3142 and PR #3281. A fix has been released in version 1.4.28.
